<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_control</class>
 <widget class="QWidget" name="form_control">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="details_editor">
       <item row="7" column="1">
        <widget class="QLineEdit" name="ln_dependancy">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="btn_dependency">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lbl_dependency">
         <property name="text">
          <string>Dependency</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_uploaders">
         <property name="text">
          <string>Uploaders</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_maintainer">
         <property name="text">
          <string>Maintainer</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lbl_architecture">
         <property name="text">
          <string>Architecture</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="lbl_descriptiontitle">
         <property name="text">
          <string>Description:  title</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="lbl_description">
         <property name="text">
          <string>body</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lbl_section">
         <property name="text">
          <string>Section</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lbl_source">
         <property name="text">
          <string>Source</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lbl_replace">
         <property name="text">
          <string>Replace</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_homepage">
         <property name="text">
          <string>Homepage</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_name">
         <property name="text">
          <string>Package Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_version">
         <property name="text">
          <string>Version</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_replace"/>
       </item>
       <item row="11" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_section"/>
       </item>
       <item row="12" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_descriptiontitle"/>
       </item>
       <item row="13" column="1" colspan="2">
        <widget class="QTextEdit" name="txt_description"/>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QComboBox" name="cb_arch">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>all</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>amd64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>i386</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_source"/>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_homepage"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_uploaders"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_maintainer">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_version"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="ln_name"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lbl_build_dep">
         <property name="text">
          <string>Build Dependency</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="ln_build_dep"/>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="btn_build_dep">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="control_file_editor">
         <item>
          <widget class="QLabel" name="lbl_control">
           <property name="text">
            <string>Control file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txt_control">
           <property name="lineWrapMode">
            <enum>QTextEdit::NoWrap</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_gencontrol">
           <property name="text">
            <string>Generate control file</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
